packages/google-cloud-compute-v1beta/google/cloud/compute/__init__.py

```diff
@@ -0,0 +1,4139 @@
+# -*- coding: utf-8 -*-
```

Comment:
This setting was not applied 
https://github.com/googleapis/googleapis/blob/8e6832c25a91a75de9024e6a42f78a8970245805/google/cloud/compute/v1beta/compute_v1beta.yaml#L110-L115

Reply:
Wait for https://github.com/googleapis/googleapis/pull/1040